src/basilisp/lang/runtime.py

```diff
@@ -275,6 +275,9 @@ def __init__(
     def __repr__(self):
         return f"#'{self.ns.name}/{self.name}"
 
+    def __call__(self, *args, **kwargs):
+        return self.value(*args, *kwargs)
```

Comment:
```suggestion
        return self.value(*args, *kwargs)  # type: ignore[not-callable]
```

I believe we'll just want to do this. We're letting users call Vars whether or not they actually are callable.

Reply:
Updated, thanks